.gitattributes

```diff
@@ -0,0 +1,11 @@
+/tests export-ignore
+/docs export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml export-ignore
+/phpunit.xml export-ignore
+/README.md export-ignore
+/CHANGELOG.md export-ignore
```

Comment:
I vote for keeping:
- README.md
- CHANGELOG.md

These are very useful to have locally and match the released version!

Reply:
No problem)

Reply:
Removed!